app/utils/lifespan.py

```diff
@@ -0,0 +1,12 @@
+from typing import AsyncContextManager
+from fastapi import FastAPI
+from app.database.database_healt_check import check_db_health
```

Comment:
You will get circular imports errors really fast with this approach
Move the lifespan funtion to service and create endpoint for /healthcheck if you want this feature, although it is outside the scope of this course

Reply:
Moved the lifespan function to service.
Created endpoint for /healthcheck